contentgrid-spring-boot-autoconfigure/src/main/java/com/contentgrid/spring/boot/autoconfigure/actuator/ActuatorAutoConfiguration.java

```diff
@@ -28,6 +29,7 @@
 
 @AutoConfiguration
 @ConditionalOnClass({PolicyActuator.class, WebHooksConfigActuator.class})
+@ConditionalOnAvailableEndpoint(endpoint = PolicyActuator.class)
```

Comment:
Doing this will also disable the webhooks actuator when only the policy actuator is disabled.

I don't think we want to be doing that?

Instead, you can create two inner classes with `@ConditionalOnAvailableEndpoint` (one for policy and one for webhook), and only the dependencies of the respective endpoints